msteams-platform/concepts/design/map-use-cases.md

```diff
@@ -82,7 +82,7 @@ The Microsoft Teams platform offers a large variety of features. Each feature is
 
 Let's look at how Teams capabilities enable different features for your app.
 
-:::image type="content" source="../../assets/images/overview/teams-apps-capabilities.png" alt-text="Image showing Teams capabilities":::
+:::image type="content" source="../../assets/images/overview/teams-apps-capabilities.png" alt-text="The screenshot is an example that shows the Teams capabilities.":::
```

Comment:
* Not a scrnshot.
* Not an example of anthing.
* Can do better than saying shows capabilities.

Reply:
Cannot add anything else in place of capabilities. The image shows the Teams capabilities.